data/altered.go

```diff
@@ -0,0 +1,85 @@
+package data
+
+// AlteredAccount is a structure that holds information about an altered account
+type AlteredAccount struct {
+	IsSender        bool
+	IsESDTOperation bool
+	IsNFTOperation  bool
+	TokenIdentifier string
+	NFTNonceString  string
+}
+
+type AlteredAccounts struct {
```

Comment:
add comments to exported stuff

Reply:
Why is this struct exported?

Reply:
refactored. this structure is not longer exported